plugins/Initial/src/Initial.ts

```diff
@@ -16,6 +16,8 @@ export interface InitialExtensions<S> {
     get(): S | undefined;
     modified(): boolean;
     unmodified(): boolean;
+    set(s: State<S>): void;
```

Comment:
Once this PR is merged, it would be good to an example to the documentation.

Reply:
:+1: